docs/API_SPECS_SIMULATIONS.md

```diff
@@ -205,6 +208,7 @@ Content-Type: application/json; charset=utf-8
 ```
 ```json
 {
+  "Name": "Sample Simulation",
```

Comment:
There is also a Description field right?

Reply:
added